### ISSUE TITLE
Select name prop isn't passed down to HiddenInput when using valueLabel

### ISSUE THREAD

**Issue by bikutadesign** [grommet/grommet#6027]
Without the `valueLabel` I'm able to retrieve the `event.target.name` value 🥳
```javascript
<Select
    name="country"
    value={value}
/>

(event) => event.target.name === 'country'
```

Without the `valueLabel` filled I'm not able to retrieve the `event.target.name` 😭
```javascript
<Select
    name="country"
    value={value}
    valueLabel={<Text>{value}</Text>}
/>

(event) => event.target.name === ''
```

Seems like the `name` prop isn't being passed down to the `HiddenInput` that's being rendered.
https://github.com/grommet/grommet/blob/master/src/js/components/Select/Select.js#L381-L387

**Comment by Ojas-Sri** [comment]
Pushed a fix in #6031

**Comment by jcfilben** [comment]
Closing since #6031 has been merged
